packages/amplify-cli/tests/amplify-helpers/headless-config.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+IFS='|'
+PROJECTNAME=\"headlessProjectName\"
+EDITOR=\"code\"
+FRONTEND=\"javascript\"
+PROVIDERS="[\"awscloudformation\"]"
+AMPLIFY="{\
+\"projectName\":$PROJECTNAME,\
+\"defaultEditor\":$EDITOR,\
+\"providers\":$PROVIDERS,\
+\"frontend\":$FRONTEND\
+}"
+REACTCONFIG="{\
+\"SourceDir\":\"src\",\
+\"DistributionDir\":\"build\",\
+\"BuildCommand\":\"npm run-script build\",\
+\"StartCommand\":\"npm run-script start\"\
+}"
+JAVASCRIPT="{\
+\"framework\":\"react\",\
+\"config\":$REACTCONFIG\
+}"
+AWSCLOUDFORMATIONPPROJECTCONFIG="{\
+\"useProfile\":false,\
+\"profileName\":\"default\",\
+\"accessKeyId\":\"headlessaccesskeyid\",\
+\"secretAccessKey\":\"headlesssecrectaccesskey\",\
+\"region\":\"us-east-1\"\
+}"
+AWSCLOUDFORMATION="{\
+\"configLevel\":\"project\",\
```

Comment:
This param doesn't make a lot of sense to the user. Should be replaced or inferred form other flags/params.

Reply:
This can be removed, and by default the configuration is done on the project level. e.g. if the user doesn't know about it, it's not going to hurt him.